File: app/src/main/java/com/fintech/scnpay/data_model/request/RequestedHistoryModel.java
package com.fintech.scnpay.data_model.request;

import com.google.gson.annotations.SerializedName;

public class RequestedHistoryModel{

	@SerializedName("DATE")
	private String dATE;

	@SerializedName("STATUS")
	private String sTATUS;

	@SerializedName("PAYMENT_MODE")
	private String pAYMENTMODE;

	@SerializedName("AMOUNT")
	private String aMOUNT;

	@SerializedName("FUND_TYPE")
	private String fUNDTYPE;

	@SerializedName("USER_ID")
	private String uSERID;

	@SerializedName("ID")
	private String iD;

	@SerializedName("TRANSACTION_ID")
	private String tRANSACTIONID;

	@SerializedName("REMARK")
	private String rEMARK;

	public String getDATE(){
		return dATE;
	}

	public String getSTATUS(){
		return sTATUS;
	}

	public String getPAYMENTMODE(){
		return pAYMENTMODE;
	}

	public String getAMOUNT(){
		return aMOUNT;
	}

	public String getFUNDTYPE(){
		return fUNDTYPE;
	}

	public String getUSERID(){
		return uSERID;
	}

	public String getID(){
		return iD;
	}

	public String getTRANSACTIONID(){
		return tRANSACTIONID;
	}

	public String getREMARK(){
		return rEMARK;
	}
}

File: app/src/main/java/com/fintech/paytcash/data_model/request/RequestedHistoryModel.kt
package com.fintech.paytcash.data_model.request

import kotlinx.parcelize.Parcelize
import android.os.Parcelable
import com.google.gson.annotations.SerializedName

@Parcelize
data class RequestedHistoryModel(

	@field:SerializedName("DATE")
	val dATE: String? = null,

	@field:SerializedName("STATUS")
	val sTATUS: String? = null,

	@field:SerializedName("PAYMENT_MODE")
	val pAYMENTMODE: String? = null,

	@field:SerializedName("AMOUNT")
	val aMOUNT: String? = null,

	@field:SerializedName("FUND_TYPE")
	val fUNDTYPE: String? = null,

	@field:SerializedName("USER_ID")
	val uSERID: String? = null,

	@field:SerializedName("ID")
	val iD: String? = null,

	@field:SerializedName("TRANSACTION_ID")
	val tRANSACTIONID: String? = null,

	@field:SerializedName("REMARK")
	val rEMARK: String? = null
) : Parcelable
